### ISSUE TITLE
Please ship IDE with LTS version of nodejs

### ISSUE THREAD

**Issue by SNathJr** [cs50/v4-issues#40]
#### What symptom did you see (attaching screenshots as necessary)?

Please ship CS50 IDE with LTS version of nodejs.

#### When date and time (UTC) did the problem happen?

5:11 pm
Friday, 28 June 2019
Coordinated Universal Time (UTC)

#### What steps did you follow to reproduce the problem?
NOT APPLICABLE
1. step 1
1. step 2
1. step 3

**Comment by dmalan** [comment]
What's the motivation? We tend to use the latest versions of most languages/runtimes installed so that students can use the latest for final projects. (And the IDE isn't designed for ongoing hosting of projects.)

**Comment by SNathJr** [comment]
Hi @dmalan. First of all, I am a huge fan of you, sir.

I tend to use the CS50 IDE for testing applications that I built on top of LTS version of nodejs. While testing the build of those applications (e.g `node install` etc.), I have found, at-least some of my project builds produce weird errors / warning messages at times (these may have been isolated incidents though). So, It was just a thought, how about using the LTS version of node instead of the latest branch. As most of us are building applications to be supported on that itself.

P.S: For the record, I am all in for having the latest versions of runtimes at my disposal.
